api/src/index.js

```diff
@@ -29,14 +27,84 @@ apiRouter.get("/", async (req, res) => {
   });
 });
 
+apiRouter.get("/all-meals", async (req, res) => {
+  try {
+    const meals = await knex.raw("SELECT * FROM Meal ORDER BY id");
```

Comment:
I know this is from last week, but maybe you could consider refactoring raw statement to using knex functions to maintain consistency in your code? :)